src/components/detail-comment/package.json

```diff
@@ -6,9 +6,9 @@
   },
   "dependencies": {
     "lodash": "4.17.21",
-    "vue": "^2.7.14"
+    "vue": "3.3.4"
```

Comment:
You no longer import Vue in this component, so this might no longer be required. Please try without it.

Reply:
Removed and worked :)